Cargo.toml

```diff
@@ -2,5 +2,8 @@
 members = [
     "ironrdp",
     "ironrdp-session",
-    "ironrdp-cli"
+    "ironrdp-ui-client",
+    "ironrdp-cli",
+    "ironrdp-renderer",
+    "ironrdp-replay-cli",
```

Comment:
What about naming it `ironrdp-replay-client`? I feel the `-cli` suffix is not necessary

Reply:
agreed.